.travis.yml

```diff
@@ -91,11 +91,20 @@ before_install:
     - pwd
     - mkdir /tmp/cozy-desktop
     - ln -s /tmp/cozy-desktop tmp
+    - npm install -g yarn
+    - cd gui
+    - yarn
+    - cd ..
 
 script:
     - npm run build
     - npm run lint
     - ./scripts/coverage.sh npm run test
+    - cd gui
+    - npm run lint
+    - npm run build
+    - npm run pack
```

Comment:
First, to know if it works, because I had absolutely no idea whether it does so 😃 

I was also wondering whether `build` could work while `pack` could fail when upgrading electron stuff. Do you think I'm just testing already tested electron tools?

Also I was thinking about having 1 (not 2) test scenario exercising the UI at some point, which would require building the app. But since it's some future plan, we don't really care.